src/components/molecules/AppNavBar/AppNavBar.tsx

```diff
@@ -13,18 +13,92 @@ import { ReactComponent as LogoFr } from '../../../assets/images/logo-horizontal
 import { FirebaseUser } from 'services/auth'
 import AccountCircle from '@material-ui/icons/AccountCircle'
 import MoreIcon from '@material-ui/icons/MoreVert'
+import Menu from '@material-ui/core/Menu'
+import MenuItem from '@material-ui/core/MenuItem'
+import Badge from '@material-ui/core/Badge'
+import MenuBookIcon from '@material-ui/icons/MenuBook'
+import Button from '@material-ui/core/Button'
 
 type Proptypes = {
   authUser: FirebaseUser
 }
 
 export default function AppNavBar({ authUser }: Proptypes) {
+  const [mobileMoreAnchorEl, setMobileMoreAnchorEl] = useState<null | HTMLElement>(null)
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null)
```

Comment:
the anchor logic is from material-ui documentation? It is the first time that I see that we use this for a menu logic.

Reply:
Yes it is basically a copy past from mu

Reply:
okay, they probably have a good reason then hihi